index.html

```diff
@@ -571,70 +571,98 @@ <h3>External Binary Content</h3>
         <blockquote id="message-external-body-variability" class="informative">
           Non-normative note: Variability among client types and locations may mean that <a>LDP-NR</a> content is
           addressed in ways that are external to the Fedora server but not resolvable by all clients. This specification
-          describes the use of <code>Content-Type: message/external-body</code> values to signal, on <code>POST</code>
-          or <code>PUT</code>, that the Fedora server should not consider the request entity to be the <a>LDP-NR</a>'s
-          content, but that a <code>Content-Type</code> value will signal a name or address at which the content might
-          be retrieved. The <code>URL</code> ([[RFC2017]]
-          <a href="https://tools.ietf.org/html/rfc2017#section-3">section 3</a>) and <code>local-file</code>
-          ([[RFC2046]] <a href="https://tools.ietf.org/html/rfc2046#section-5.2.3.4">section 5.2.3.4</a>) values of the
-          <code>access-type</code> parameter motivate this specification, but a Fedora server may support any
-          <code>access-type</code> parameters per the requirements for advertisement and rejection specified here.
+          describes the use of a <code>rel="http://fedora.info/definitions/fcrepo#ExternalContent"</code> link in a
+          <code>Link</code> header to signal, on <code>POST</code> or <code>PUT</code>, that the Fedora server should
+          not consider the request entity to be the <a>LDP-NR</a>'s content, but instead that the target of the
+          link gives the URI from which the content might be retrieved. The <code>http(s):</code> [[RFC3986]] and
+          <code>file:</code> [[RFC8089]] URI schemes motivate this specification, but a Fedora server may support
+          other URI schemes for addressing external content per the requirements for advertisement and rejection
```

Comment:
I don't believe we specify a mechanism for advertising the URI schemes — should we add an `Accept-External-Content-Scheme` header to `OPTIONS` responses for this?  Or advertise it some other way?

Reply:
True. It seems to me that knowing about support for `file:` as a scheme will never be enough to use it, so advertising support doesn't really matter. However, general support for `http` and/or `https` would be usable without additional information. Off the top of my head I don't see anything better than `Accept-External-Content-Scheme: file,http,https` as you suggest

Reply:
I note that with `message/external-body` there was a way to advertise `file` or `anything-else` but not to distinguish between `http`, `https`, `ftp` etc..